config/locales/idv/en.yml

```diff
@@ -21,6 +21,7 @@ en:
       ccn: Last 8 digits of a credit card
       city: City
       dob: Date of birth
+      dob_hint: 'example: 07/04/1999'
```

Comment:
I always like `12/31/1969` but then, I'm an old grey unix nerd.

Maybe `01/17/1964` -- I'll let you figure out why.